### ISSUE TITLE
Failed install on node.js

### ISSUE THREAD

**Issue by fcasarsasolari** [indexeddbshim/IndexedDBShim#254]
Hello,

as in the documentation i tryed installing using node:
`npm install indexeddbshim` but i can not use the the library.

I can not even to load it with require, failing with a ReferenceError: window is not defined ( that is quite obvious). Looking at the downloaded files in node_modules i don't see the indexeddbshim-node.*.js files.

Thanks.

**Comment by brettz9** [comment]
The Node support is only in `master` and not yet part of a published release. If you want to use in Node, for now, you'd have to clone the repo. I'll close, but feel free to post any further comments if you wish.

**Comment by chrislgrigg** [comment]
Has any thought been given to releasing the version with node support to NPM?

**Comment by brettz9** [comment]
See my comment at https://github.com/axemclion/IndexedDBShim/issues/259#issuecomment-248397722

**Comment by brettz9** [comment]
There had been an issue limiting Node use now fixed in `master` (as discussed in #262).

For npm support, pending a stable version, you can use `npm`'s ability to directly import Git repositories like: 
```json
"dependencies": {
    "indexeddbshim": "https://github.com/axemclion/IndexedDBShim"
  },
```

or, refer to a specific tag, SHA commit or branch as per https://docs.npmjs.com/files/package.json#git-urls-as-dependencies. But still need to get more done before a bona fide versioned release on our side I think.
